common/src/main/java/com/redhat/cloud/notifications/models/dto/v1/endpoint/properties/secrets/BasicAuthenticationDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.redhat.cloud.notifications.models.dto.v1.endpoint.properties.secrets;
+
+public final class BasicAuthenticationDTO {
+    private String username;
+    private String password;
+
+    public BasicAuthenticationDTO(String username, String password) {
```

Comment:
Do we need that constructor? Bonus question: shouldn't Jackson complain that no default constructor is available when it needs to build an instance of this class?

Reply:
No, we don't need it! Removed!